webpack/plugins/tauri-require.js

```diff
@@ -0,0 +1,59 @@
+module.exports.plugin = class TauriRequirePlugin {
+  apply (compiler) {
+    compiler.plugin('compilation', function (compilation) {
+      compilation.mainTemplate.hooks.requireEnsure.tap('JsonpMainTemplatePlugin load', (source, chunk, hash) => {
+        return `
+          // Tauri chunk loading
+
+          var installedChunkData = installedChunks[chunkId];
```

Comment:
are you linting? there's a lot of stuff here that I don't like
var, ;

Reply:
I just adapted it from webpack's code